contracts/Nexus.sol

```diff
@@ -551,6 +563,17 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
         else revert UnsupportedExecType(execType);
     }
 
+    /// @dev Executes a single transaction based on the specified execution type.
+    /// @param executionCalldata The calldata containing the transaction details (target address, value, and data).
+    /// @param execType The execution type, which can be DEFAULT (revert on failure) or TRY (return on failure).
+    function _handleDelegateCallExecution(bytes calldata executionCalldata, ExecType execType) private {
+        address delegate = address(uint160(bytes20(executionCalldata[0:20])));
+        bytes calldata callData = executionCalldata[20:];
```

Comment:
can use the library method from above again

Reply:
done